docs/core/tutorials/cli-templates-create-template-pack.md

```diff
@@ -98,19 +98,27 @@ The `<ItemGroup>` contains two settings. First, the `<Content>` setting includes
 
 ## Build and install
 
-Save this file and then run the pack command
+Save the project file. Before building the template pack, verify that your folder structure is correct. Any template you want to pack should be placed in the _templates_ folder, in its own folder. The folder structure should look similar to the following:
 
-```dotnetcli
-dotnet pack
+```console
+working
+|   templatepack.csproj
```

Comment:
Used `│` instead of `|`. (later lines already uses `│`

```suggestion
│   templatepack.csproj
```

Reply:
I figured one may have snuck in there. I'm waiting on build to finish so I can see the preview. Thanks!